services/garage-backend/src/models/Thing.ts

```diff
@@ -12,6 +12,7 @@ export interface ThingDocument
 const thingSchema = new mongoose.Schema<ThingDocument>({
   name: {
     type: String,
+    index: { unique: true },
```

Comment:
hmm, are you sure this is the way field is made unique? 
```suggestion
    unique: true,
```
I'm quite sure it's just `unique`